docs/validators/sigstore_cosign.md

```diff
@@ -144,10 +144,22 @@ The secret can for example be created directly from your local `config.json` (fo
 ```bash
 kubectl create secret generic my-secret \
   --from-file=.dockerconfigjson=path/to/config.json \
-  --type=kubernetes.io/dockerconfigjson -n connaisseur
+  -n connaisseur
```

Comment:
@funkypenguin was there a specific reason for this change or did it cause any errors? I noticed that dropping this line works with connaisseur, but means that the secret cannot anymore be used as an `imagePullSecret` since those need to have the `type` specified, see e.g. [here](https://kubernetes.io/docs/tasks/configure-pod-container/pull-image-private-registry/#registry-secret-existing-credentials).
If that is not the case, I will re-introduce it in #585

Reply:
It didn't seem necessary, so I removed it for simplicity. I didn't realize that the intention was to reuse it as an imagePullSecret in the connaisseur namespace, so if that's the case then it should be reintroduced :)

Reply:
let's say it could be re-used as an image pull secret or at least has the same form which probably makes it easier to work with. will fix :wink: